VerilogCodingStyle.md

```diff
@@ -797,6 +797,76 @@ endmodule : foo
 localparam bit ValBaz = 1;  // This comment describes the item to the left.
 ```
 
+It can sometimes be useful to structure the code using header-style comments in
+order to separate different functional parts (like FSMs, the main datapath or
+registers) within a module. In that case, the preferred style is a single-line
+section name, framed with `//` C++ style comments as follows:
+
+```systemverilog
+module foo;
+
+  ////////////////
+  // Controller //
+  ////////////////
+  ...
+
+  ///////////////////////
+  // Main ALU Datapath //
+  ///////////////////////
+  ...
+
+  //////////////////////////////////////////////////////////////////////////////
+  //        It is ok to play with the width of the comment like this          //
```

Comment:
I don't see any advantage in allowing this variant. It doesn't save people work, it reduces uniformity, and it just opens up the possibility of the text being misaligned etc.

Reply:
ok good I restrict it to one single style.